va_explorer/va_data_management/views.py

```diff
@@ -68,8 +69,9 @@ def get_context_data(self, **kwargs):
         return context
 
 
```

Comment:
@smacadam Reset and RevertLatest have to be similarly restricted

Reply:
added permissions and tests